modules/bmo-getting-the-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,136 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+:_content-type: PROCEDURE
+[id="getting-the-baremetalhost-resource_{context}"]
+= Getting the BareMetalHost resource
+
+The `BareMetalHost` resource contains the properties of a physical host. You must get the `BareMetalHost` resource for a physical host to review its properties.
+
+.Procedure
+
+. Get the `BareMetalHost` resource for all hosts:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api -o yaml
+----
++
+[NOTE]
+====
+You can use `baremetalhost` as the long form of `bmh` with `oc get` command.
+====
+
+. Get the list of hosts:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
+
+. Get the `BareMetalHost` resource for a specific host
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api/<host_name> -o yaml
```

Comment:
Please, change the command to `oc get bmh <host-name> -n openshift-machine-api -o yaml`. In the format that is given here the user will receive an error like error: invalid namespace "openshift-machine-api/openshift-worker-0-1": [may not contain '/']